Several typo fixes and improvements of the Android architecture images (#336)
memfault/memfault-docs: docs/android/android-getting-started-guide.mdx

feature in the top right hand side of the `Issues` page. This shows a list of
bug reports that were recently received and their processing status.

> A bug report's status would stay `Stored` if the maximum number of bug reports

Reviewer: ```suggestion
A bug report's status will remain as `Stored` if the maximum number of bug reports
```

that will be analyzed from a device per day is reached. Quotas can be accessed
from

> [Settings -> Quotas](https://app.memfault.com/organizations/-/projects/-/settings/quotas).

Reviewer: I think we do have an MDX component for these links. But also, should we rather link to the docs page about quotas?

Reviewer 2: I would link to both, since whenever we say `App Section -> Some Tab` we usually link to the app as well.

We don't have an MDX component, but we have a JSX component. It's a bit cumbersome to use because it needs to be used in HTML, so you'd have to change this paragraph to HTML.

I've been wanting to figure out a way to have a markdown/Docusaurus extension to use it in Markdown, but ey.

It's called `ProjectMagicLink`.

Author: Added links to the docs too :)

> When a bug report is `Processed`, results of analysis can be viewed from the

Reviewer 2: Nit, probably a matter of taste:

```suggestion
When a bug report is `Processed`, the results of analysis can be viewed from the
```
